CMakeLists.txt

```
@@ -236,3 +236,17 @@ if(SDFLIB_BUILD_DEBUG_APPS)
     # target_link_libraries(ImageQueryTime PUBLIC ${PROJECT_NAME} CGAL::CGAL OpenVDB::openvdb)
     target_link_libraries(ImageQueryTime PUBLIC ${PROJECT_NAME})
 endif()
+
+install(TARGETS ${PROJECT_NAME} icg EXPORT ${PROJECT_NAME}Config
+    RUNTIME DESTINATION bin
+    ARCHIVE DESTINATION lib
+    LIBRARY DESTINATION lib
+)
+
+install(EXPORT ${PROJECT_NAME}Config DESTINATION share/sdflib)
```

For recommend CMake practice, the export should add a namespace.
```suggestion
install(EXPORT ${PROJECT_NAME}Config
    NAMESPACE ${PROJECT_NAME}::
    DESTINATION share/sdflib
)
```
leading to 
~~~cmake
find_package(SdfLib CONFIG REQUIRED)
target_link_libraries(mine PRIVATE SdfLib::SdfLib)
~~~

---

I can add namespace, but then you want me to add alias as well